### PR TITLE
release: v1.1.0-rc.1

### PR DESCRIPTION
**Pull request by guydc**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Release v1.1.0-rc.1

### CODE REVIEW

File: release-notes/v1.1.0-rc1.yaml [hunk 1]
```diff
@@ -0,0 +1,229 @@
+date: July 8, 2024
+
+changes:
+  - area: documentation
+    change: |
+      Added Performance Benchmarking Document
+      Added User Guide for Zipkin Tracing
+      Added User Guide for Customizatio of Filter Order
```
Comment (arkodg) [inline release-notes/v1.1.0-rc1.yaml:8]:
```suggestion
      Added User Guide for Customizating for Ordering Filters
```

File: release-notes/v1.1.0-rc1.yaml [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      Fixed duplication of Ext-Auth, OIDC and Basic Auth Filters
+      Fixed Proxy Protocol Filter to always be the first Listener Filter
+      Fixed Translation Consistency by sorting Gateways
+      Fixed QUIC Listener to only Advertise HTTP/3 over ALPN
+      Fixed SNI matching for TCP Routes with TLS termination
+      Fixed Reconciliation when EnvoyProxy backendRefs changes
+      Fixed Reconciliation when a referenced Secret or ConfigMap changes
+      Fixed ReplaceFullPath not working for root path
+      Fixed Default Application Protocol to TCP
+      Fixed not using well-known ports (80, 443) in rediret location
```
Comment (arkodg) [inline release-notes/v1.1.0-rc1.yaml:211]:
```suggestion
      Fixed not appending well-known ports (80, 443) in redirect Location header
```

File: release-notes/v1.1.0-rc1.yaml [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+      Fixed BackendTLSPolicy status to fully conform with PolicyStatus
+      Fixed duplication of Ext-Auth, OIDC and Basic Auth Filters
+      Fixed Proxy Protocol Filter to always be the first Listener Filter
+      Fixed Translation Consistency by sorting Gateways
+      Fixed QUIC Listener to only Advertise HTTP/3 over ALPN
+      Fixed SNI matching for TCP Routes with TLS termination
+      Fixed Reconciliation when EnvoyProxy backendRefs changes
+      Fixed Reconciliation when a referenced Secret or ConfigMap changes
+      Fixed ReplaceFullPath not working for root path
+      Fixed Default Application Protocol to TCP
```
Comment (arkodg) [inline release-notes/v1.1.0-rc1.yaml:210]:
```suggestion
      Fixed Default Application Protocol to TCP for Zipkin Tracing
```

File: release-notes/v1.1.0-rc1.yaml [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+      Fixed JSON Access Log Example
+      Use linkinator to detect dead links in docs
+      Use helm-docs to generate chart docs
+      Support Not-Implemented-Hide marker in API docs
+
+
+  - area: installation
+    change: |
+      Added new gateway-addons-helm chart for Observability
+      Added support for global image settings for all images in the EG chart
```
Comment (arkodg) [inline release-notes/v1.1.0-rc1.yaml:62]:
```suggestion
      Added support for global image settings for all images in the Envoy Gateway helm chart
```

File: release-notes/v1.1.0-rc1.yaml [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+      Added Support for Loadbalancing Header Hash Policy in BackendTrafficPolicy CRD
+      Added Support for Cluster Connection Buffer Size Limit in BackendTrafficPolicy
+      Added Support for more Rate Limit Rules in BackendTrafficPolicy CRD
+      Added Support for WASM extension in EnvoyExtensionPolicy CRD
+      Added Support for External Processing extension in EnvoyExtensionPolicy CRD
+      Removed Status Print Column from xPolicy CRDs
+
+
+    breaking-change: |
+      Gateway-API BackendTLSPolicy v1alpha3 is incompatible with previous versions of the CRD
```
Comment (arkodg) [inline release-notes/v1.1.0-rc1.yaml:132]: we need to add some Upgrade notes in here similar to https://gateway-api.sigs.k8s.io/guides/?h=release+not#backendtlspolicy

Comment (guydc) [reply release-notes/v1.1.0-rc1.yaml:132]: Any thoughts on how to include this? Just add a reference to the GW-API docs in this comment? Another "Upgrade Guide" section in the release notes? should we create a proper Upgrade Guide doc for each release?

**Review by arkodg — APPROVED** [review]
LGTM thanks !

File: release-notes/v1.1.0-rc1.yaml (hunk 5 above)
Comment (arkodg) [reply release-notes/v1.1.0-rc1.yaml:132]: thanks for adding it to the Quickstart, can we also add the  link in here ?

**Review by shawnh2 — APPROVED** [review]

**Review by Xunzhuo — COMMENTED** [review]
I think we should generate the release note based the yaml.

**Pull request merged**